whc-shared/src/main/java/com/ibm/whc/deid/shared/pojo/config/masking/DateTimeConsistentShiftMaskingProviderConfig.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.whc.deid.shared.pojo.config.masking;
+
+import java.time.format.DateTimeFormatterBuilder;
+import java.util.List;
+import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.ibm.whc.deid.shared.pojo.config.DeidMaskingConfig;
+import com.ibm.whc.deid.shared.pojo.masking.MaskingProviderType;
+import com.ibm.whc.deid.shared.util.InvalidMaskingConfigurationException;
+
+@JsonInclude(Include.NON_NULL)
+public class DateTimeConsistentShiftMaskingProviderConfig extends MaskingProviderConfig {
+
+  private static final long serialVersionUID = 5266193758403530535L;
+
+  /**
+   * Determines where the shifted datetime value is ordered in relation to the original datetime
+   * value.
+   */
+  public static enum DateShiftDirection {
+
+    /**
+     * The shifted value is before the original value.
+     */
+    before,
+
+    /**
+     * The shifted value can be before or after the original value.
+     */
```

Comment:
I think we usually define enum in upper case? If we want the value to be displayed as lower case, we can use annotations(@JsonProperty?)

Reply:
changing